doc/getting-started.md

```diff
@@ -1,7 +1,5 @@
 # Getting started with Istio
 
-## This document is <span style="color:red">UNDER CONSTRUCTION</span>
-
 ## Before you begin
 
 This tutorial assumes you have a working Kubernetes of at least version 1.5.2.  Do `kubectl version` to verify
```

Comment:
I am not sure what you mean.  I based the writing style on https://kubernetes.io/docs/tutorials/connecting-apps/connecting-frontend-backend/ and on _istio/istio/demos/apps/bookinfo/README.md_.  Those docs use "we" and "you" all over the place.